src/manage_sql_tls_certificates.c

```diff
@@ -192,9 +231,33 @@ tls_certificate_count (const get_data_t *get)
 {
   static const char *filter_columns[] = TLS_CERTIFICATE_ITERATOR_FILTER_COLUMNS;
   static column_t columns[] = TLS_CERTIFICATE_ITERATOR_COLUMNS;
+  gchar *filter;
+  char *extra_where;
+  int ret;
+
+  if (get->filt_id && strcmp (get->filt_id, FILT_ID_NONE))
```

Comment:
Would be great if we could get this filter setup into a function at some point.

Reply:
I agree, this kind of filter handling is a lot of duplicated code that should be in a function. 
Maybe we could also get rid of the `filter ? filter : get->filter` by updating `get->filter` in `get_init()`.

Reply:
Yeah, that might be a good idea.